composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "minimum-stability": "stable",
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
+        "phpunit/phpunit": "4.6.*",
```

Comment:
I prefer not to bother composer with PHPUnit, similar to `composer self-update`.